docs/browser-engine/electron.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Electron
+---
+
+# Electron
+
+Electron is a popular framework to build cross-platform desktop applications using web
+technologies. Some of the most popular applications using it are [Visual Studio Code], [Slack],
+[Discord], [InVision], [Figma], or [WhatsApp].
+
+While it was originally created by GitHub in 2013, it is an [OpenJS Foundation][electron-foundation]
+project since 2019. It has an [Open Governance] where engineers from different companies and
+products (like VS Code, Slack, Teams, RingCentral) and volunteers collaborate in different [Working
+Groups].
+
+Electron applications use Chromium as the shell to render the User Interface. Developers
+use HTML, CSS and JavaScript to build their apps, which means they can use the libraries and tools
+available to the web community such as React, webpack, Babel, etc. One of the main differences with
+other technologies under the _[browser-engine]_ category, is that JavaScript is used to access the
+Operating System features as well. This is because Electron applications come with [Node.js]
+support. At a high-level, [Electron adds the Node.js layer to Chromium's JavaScript engine
+V8][electron-node]. This means that developers can leverate all its ecosystem and also write C/C++
+code to access features not available out of the box or via other modules.
+
+## Release cadence, version support and update model
+
+Electron is built on top of Chromium and releases every other version of Chrome. This means
```

Comment:
~~Electron is built on top of Chromium and releases every other version of Chrome~~

Electron has three important branches of development. The nightly branch is synced to Chromium's development branch and merged weekly or daily. Nightly electron is equivalent to Chrome Canary. The beta and stable branches of Electron are fixed to their major-versions of Chromium and release every 12 weeks. New Electron stable versions typically release the same day as the equivalent stable Chrome release.